.travis.yml

```diff
@@ -106,6 +106,7 @@ jobs:
     - sudo /snap/bin/lxd.migrate -yes
     - sudo /snap/bin/lxd waitready
     - sudo /snap/bin/lxd init --auto
+    - git fetch --tags
```

Comment:
snap was failing on master and PR builds. Not sure how this is related.

Reply:
I think travis was failing in some cases to pull tags, so `git describe` was failing, meaning `snapcraft` couldn't get a version.